Training2/thongnq-training2/src/App.js

```diff
@@ -0,0 +1,89 @@
+import React, {useState,useEffect, useRef} from "react";
+
+import Button from "./task3/Button";
+
+function App() {
+
+  const [count, setCount] = useState(0);
+  const [startInt, setStartInt] = useState(false);
+  const [intervalId, setIntervalId] = useState(0);
```

Comment:
@Thong-Nguyen-Zigvy  you should not useState to store interval because it can make the component re-render, you can clear the invertal inside the 2nd useEffect with clean effect function